Remove PNS ID (#167)
quicwg/multipath: draft-ietf-quic-multipath.md

- Path Identifier (Path ID): An identifier that is used to identify
  a path in a QUIC connection at an endpoint. Path Identifier is used

>   in multipath control frames (etc. PATH_ABANDON frame) to identify

Reviewer: ```suggestion
  in multipath control frames (e.g., PATH_ABANDON frame) to identify
```

  a path. The initial path that is used during the handshake (and multipath negotiation)
  has the path ID 0 and therefore all 0-RTT packets are also tracked and

>   processed with the path ID 0. For 1-RTT packets, the path ID is the 

Author: ```suggestion
  processed with the path ID 0. For 1-RTT packets, the path ID is the
```

The ACK_MP frame, as specified in {{ack-mp-frame}}, is used to
acknowledge 1-RTT packets.
Compared to the QUIC version 1 ACK frame, the ACK_MP frame additionally
contains a Path ID.

> The Path ID used to distinguish packet number spaces for different

Reviewer: ```suggestion
The Path ID is used to distinguish packet number spaces for different
```
